portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,9 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+function getGreetingUsingArrowFunctions() {
+  fetch('/data').then(response => response.text()).then((greeting) => {
+    document.getElementById('greeting-container').innerText = greeting;
+  });
+}
```

Comment:
Tiny nit: if you add a new line at the very end of this file (just after the `}`) that little red icon will go away

Reply:
Addressed in 3.5part1